src/test/groovy/org.betterdevxp.gradle.testkit/ProjectValidator.groovy

```diff
@@ -66,6 +67,8 @@ class ProjectValidator {
                 return it
             } else if (it instanceof Task) {
                 return it.name
+            } else if (it instanceof TaskProvider) {
```

Comment:
could this condition be combined with the `if` above with an `||`? i.e. would there ever be something different returned for `TaskProvider`?

Reply:
updated